programmers/creat-strange-characters.js

```diff
@@ -0,0 +1,25 @@
+function solution(s) {
+    let altrnChar; 
+    let altrnCollection;
```

Comment:
let은 초기값을 안써도 되지만 써주시는 것이 좋습니다.
배열은 const로 선언해주세요
```suggestion
    const altrnCollection = [];
    let altrnChar = ""; 
```